posthog/models/session_recording_event/util.py

```diff
@@ -18,3 +18,27 @@ def get_recording_count_for_team_and_period(
         {"team_id": str(team_id), "begin": begin, "end": end},
     )[0][0]
     return result
+
+
+def get_recording_count_month_to_date() -> int:
+    result = sync_execute(
+        """
+        -- count of recordings month to date
+        SELECT count(distinct session_id) as freq
+        FROM session_recording_events
+        WHERE toStartOfMonth(timestamp) = toStartOfMonth(now());
+    """
+    )[0][0]
+    return result
+
+
+def get_recording_events_count_month_to_date() -> int:
+    result = sync_execute(
+        """
+        -- count of recordings events month to date
```

Comment:
This sort of comment makes it impossible to copy the query in metabase. Let's remove it.